Changelog v1.8.x.md

```diff
@@ -23,6 +24,10 @@
 - Fix: Textures of Universal Block Modifier, Universal Owner Changer and the Mine Remote Access Tool are transparent at the sides (Thanks Redstone_Dubstep!)
 - Fix: Step sounds of disguised blocks are wrong
 - Fix: Camera Monitor tooltip shows amount of first X bound cameras instead of amount of all bound cameras
+- Fix: Mines and Portable Radar exist without bottom support block
+- Fix: Claymore placement is inverted
```

Comment:
This change is simply wrong. The claymore should be facing away from the player when they place it. It's more realistic.

Reply:
The same realistic as it is that the claymore ignores its owner? In my opinion, either the claymore respects its owner and the placement is not neccessary, or the claymore doesn't and then yes, it would be rather stupid that you place a detection mine that detects everything in your direction and instandly detonates. Please tell me your opinion though, you are the dev

Reply:
Let's put it a different way: From a gameplay point of view (ignoring realism), it doesn't matter how the claymore is placed. Thus making this change is unnecessary as it has been a certain way for years. A change would just mean that players who are used to the one way have to adapt to the new way without having a profound reason why.

Reply:
Ok. I'm sorry for having done this change, but I'm not into mines that much and it felt for me like it was a bug, because normally the blocks look in your direction when placing them.

Reply:
No need to be sorry, this is why we can discuss propsed changes :)